CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+
+- Added `signInWithNewPassword` GraphQL API for signing in with a new password.
+
 ### Changed
 
```

Comment:
In line with this [comment](https://github.com/aicers/review-web/pull/291#issuecomment-2449076264), I noticed that the `signIn` API logic has been modified. As this logic impacts end users, I think it seems beneficial to explicitly include this change in the changelog to keep our users informed. I’d suggest adding a line in the Changed section, perhaps like the following:

- The `signIn` GraphQL API now returns an error if the account was never signed in before. This change is part of enhancing account security by requiring all users to update their passwords upon their first signing in.